tests/components/dunehd/test_config_flow.py

```diff
@@ -67,7 +67,7 @@ async def test_user_invalid_host(hass):
 async def test_user_very_long_host(hass):
     """Test that errors are shown when the host is longer than 253 chars."""
     long_host = (
-        "very_long_host_very_long_host_very_long_host_very_long_host_very_long"
+        "very_long_host_very_long_host_very_long_host_very_long_host_very_long_"
         "host_very_long_host_very_long_host_very_long_host_very_long_host_very_long_ho"
         "st_very_long_host_very_long_host_very_long_host_very_long_host_very_long_host_"
```

Comment:
As you touched it anyway, it can be improved a bit further 🙂
```suggestion
        "host_very_long_host_very_long_host_very_long_host_very_long_host_very_long_"
        "host_very_long_host_very_long_host_very_long_host_very_long_host_very_long_host_"
```